docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,102 @@ keywords:
   - temporal cloud nexus
 ---
 
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
 :::tip SUPPORT, STABILITY, and DEPENDENCY INFO
 
-Temporal Nexus is available in [Public Preview](https://docs.temporal.io/evaluate/development-production-features/release-stages#public-preview) for both self-hosted deployments and Temporal Cloud with the [Temporal Go SDK](https://docs.temporal.io/develop/go).
+**Temporal Nexus** is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for [Temporal Cloud](/cloud/nexus) and [self-hosted deployments](/production-deployment/self-hosted-guide/enable-nexus).
 
 :::
 
-Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries.*
-It promotes a more modular architecture for sharing a subset of your team’s capabilities through well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, like Workflows, or execute arbitrary code.
+## Connect Temporal Applications
+
+Nexus allows you to reliably connect Temporal Applications.
```

Comment:
Maybe something like: 

Nexus allows you to reliably connect Temporal Applications, , helping teams create reliable, durable, and modular services. 

This helps me understand why I would want to do so.

Reply:
I would personally make this a "what exactly is Nexus" section if you want to focus on understanding and then move lines 31-36 to under "Why use Nexus". 

I think it's good to have an introduction that clarifies what it is first before letting users know what it is in it for them.

Reply:
I took a shot at this up above:

"Nexus allows you to connect Temporal Applications across isolated namespaces. This provides all of the benefits of Durable Execution across team and application boundaries for improved modularity, security, troubleshooting, and fault isolation."

Reply:
> I would personally make this a "what exactly is Nexus" section if you want to focus on understanding and then move lines 31-36 to under "Why use Nexus".
> I think it's good to have an introduction that clarifies what it is first before letting users know what it is in it for them.

Thanks, I like the suggestion to create a "What is Nexus" section and cleaning up the "Why use Nexus", and your bullet list suggestions below. Will add those and incorporate additional content ☝️ that is a riff on your "why nexus" suggestions below.

However, would like to keep this self-contained intro/elevator pitch section though to provide the gist of Nexus without reading the rest of the evaluate content. We received feedback the intro section was too lengthy for some audiences, so have shortened it up to a self-contained elevator pitch. The title was selected to be the essential "what and why" behind Nexus - to connect Temporal Applications.